benchmarks/spidermonkey/sightglass.h

```diff
@@ -1 +0,0 @@
-../../include/sightglass.h
```

Comment:
We probably want to keep the symlink in case we change this header in the future? Or are there changes to this file?

Reply:
The docker build was failing with the symlink. No idea why!